src/main/java/pl/company/foodatu/plans/domain/InMemoryAvailableMealsRepository.java

```diff
@@ -0,0 +1,25 @@
+package pl.company.foodatu.plans.domain;
```

Comment:
In fact, we can treat in memory adapters as part of infrastructure

Reply:
But InMemoryAvailableMealsRepository class uses AvailableMeal class, which is package private :/

Reply:
`AvailableMeal` became part of our API, we can make it public